app/views/events/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%= @event.name %></h1>
+<p>Created by: <%= @event.creator.name %> </p>
+<p>Date: <%= @event.date %></p>
+<p>Location: <%= @event.location %></p>
+<p><%= @event.description unless @event.description.nil? %></p>
+<br>
+
+<%= link_to "I'm attending!", attendance_path(@event.id), method: :post unless (current_user.nil? || user_attending) %>
+
+<% if !@event.event_attendees.empty? %>
+  <h3>Attendees:</h3>
+  <ul>
+  <% @event.event_attendees.each do |attendee| %>
```

Comment:
Store `event_attendees` in an instance variable in the controller so you avoid calling it twice from the view